nemo/export/trt_llm/nemo_ckpt_loader/nemo_file.py

```diff
@@ -395,7 +395,7 @@ def load_nemo_model(nemo_ckpt: Union[str, Path], nemo_export_dir: Union[str, Pat
                         nemo_model_config[k] = v
                     elif k == "activation_func":
                         if isinstance(v, torch.jit.ScriptFunction):
-                            nemo_model_config["activation"] = v.name.replace("_", "-")
+                            nemo_model_config["activation"] = v.name
```

Comment:
n00b question: could you explain the reasoning for this change? the PR doesn't have any description and it's hard to parse out from this alone. are there any tests that exercise this path?